lib/publiccloud/provider.pm

```diff
@@ -360,7 +360,7 @@ sub terraform_apply {
         $cmd .= "-var 'region=" . $self->region . "' ";
         $cmd .= "-var 'name=" . $name . "' ";
         $cmd .= "-var 'project=" . $args{project} . "' " if $args{project};
-        $cmd .= sprintf(q(-var 'tags={"openqa_ttl":"%d"}' ), get_var('MAX_JOB_TIME', 7200) + 300);
+        $cmd .= sprintf(q(-var 'tags={"openqa_ttl":"%d"}' ), get_var('PUBLIC_CLOUD_DEFAULT_TTL', 7500));
```

Comment:
we need to consider `MAX_JOB_TIME` as some jobs need to increase this see: https://openqa.suse.de/tests/4198854#settings

```suggestion
        $cmd .= sprintf(q(-var 'tags={"openqa_ttl":"%d"}' ), get_var('PUBLIC_CLOUD_DEFAULT_TTL', get_var('MAX_JOB_TIME', 7200) + 300));
```

Reply:
I edited my last comment. So `PUBLIC_CLOUD_DEFAULT_TTL` is used if _no_ MAX_JOB_TIME is set.

Reply:
Well job which you mention just showing to me that we need to inform @pdostal about this PR and ask him to add to his tests `PUBLIC_CLOUD_DEFAULT_TTL=18000` variable . 
Because for me main idea is actually to **remove** use of `MAX_JOB_TIME` in this context

Reply:
The job takes longer then the default 2h. So as I understood it, it is mandatory to set the `MAX_JOB_TIME`. 
Your intention is a complete different thing then `MAX_JOB_TIME` from my point of view. You would like to keep VM's longer up, then the corresponding openqa job, to get some debug capabilities... or?

Reply:
Maybe you should use `$ttl = max(PUBLIC_CLOUD_DEFAULT_TTL, MAX_JOB_TIME);`

Reply:
or
```suggestion
        $cmd .= sprintf(q(-var 'tags={"openqa_ttl":"%d"}' ), get_var('MAX_JOB_TIME', 7200) + get_var(PUBLIC_CLOUD_TTL_OFFSET, 300));
```

Reply:
This is OK for me, thank you for letting me know @asmorodskyi